src/Sulu/Bundle/PageBundle/Domain/PageInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PageBundle\Domain;
+
+interface PageInterface
+{
+    const RESOURCE_KEY = 'pages';
```

Comment:
we cant really use this interface anywhere, right? think i would use the `BasePageDocument` to store the constant for now

Reply:
fixed